index.html

```diff
@@ -146,25 +245,32 @@ <h5>About me</h5>
         </div>
     </section>
     <footer>
-        <h1>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
-        <form action="https://formspree.io/f/mlevqbqb" method="post"> 
-            <ul>
-                <li>
-                    <input name="user_name" type="text" id="name" maxlength="30" value="Full name" required />
-                </li>
-                <li>
-                    <input name="user_mail" type="email" id="email" value="Email" required />
-                </li>
-                <li>
-                    <textarea name="user_msg" id="message" cols="30" rows="10" maxlength="500" required>Enter text here</textarea>
-                </li>
-                <li>
-                    <button class="submit-btn" type="submit">Get in touch</button>
-                </li>
-            </ul>
-        </form>
-        <hr class="solid">
-        <div class="social-media-icon">
+        <div class="form-form">
+            <div class="interest">
+                <h1>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            </div>
+            <div class="form-inputs">
+                <form action="https://formspree.io/f/mlevqbqb" method="post">
+                    <ul class="form-list">
+                        <li>
+                            <input name="user_name" type="text" id="name" maxlength="30" placeholder="Name" required />
+                        </li>
+                        <li>
+                            <input name="user_mail" type="email" id="email" placeholder="Email address" required />
+                        </li>
+                        <li>
+                            <textarea name="user_msg" id="message" cols="20" rows="2" maxlength="500" placeholder="Hello Rita. Glad to see you |"
+                                required></textarea>
+                        </li>
+                        <li>
+                            <button class="submit-btn" type="submit">Get in touch</button>
+                        </li>
+                    </ul>
+                </form>
+            </div>
+        </div>
+        <div class="social-media-icon-2">
```

Comment:
- [ ] Kindly make sure that you use the appropriate color for the icons `background: #505F79;` It will make your icons look like it is in the Figma design as required in the project :+1: 

- [ ] [ Optional] Kindly I can suggest you when you design your desktop version please make sure does not affect the mobile version. please fix the arrangement of `social-media-icon-2` of the contact form in the mobile version it will make your design stick much to the Figma design :+1:  :+1: 

## Figma design 
![image](https://user-images.githubusercontent.com/95297251/206576472-933c018a-216f-4321-95ec-1808218e601d.png)


## Your design 

![image](https://user-images.githubusercontent.com/95297251/206575321-3b4a9f6c-659d-4af7-80e8-4114de600925.png)